src/main/java/com/emonadeo/autorun/MovementDirection.java

```diff
@@ -16,28 +16,28 @@ public enum MovementDirection {
     public KeyBinding getKeyBinding(MinecraftClient client) {
         switch (this) {
             default:
-                return client.options.keyForward;
```

Comment:
These changes are required due to the option changes described [here](https://fabricmc.net/2022/06/07/119.html)